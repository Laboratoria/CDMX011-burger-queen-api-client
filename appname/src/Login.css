@import url("https://fonts.googleapis.com/css2?family=Ropa+Sans&display=swap");

/* http://meyerweb.com/eric/tools/css/reset/ 
   v2.0 | 20110126
   License: none (public domain)
*/

* {
    margin: 0;
    padding: 0;
}

body {
  font-family: "Ropa Sans", sans-serif;
  background-color: #ffeaae;
  color: #673500;
  height: 100vh;
}

main {
  height: 100vh;
  display: flex;
}

aside {
  background-color: rgba(255, 193, 0, 0.71);
  width: 35%;
  font-size: 2em;
}

aside svg {
  width: 80%;
  margin: 0 10%;
}

aside h1 {
  text-align: center;
  font-size: 1.5em;
}

aside,
.login {
  display: flex;
  flex-direction: column;
  justify-content: center;
}

.login {
  width: 65%;
    margin: 0;
    gap:1.5em;
}

.login h2{
    font-size:2.5em;
    text-align: center;
}

.login form {
  height: 50%;
  width: 50%;
  background-color: #ffc100;
  box-sizing: border-box;
  margin: 0 auto;
  border: 8px solid #802d10;
  box-shadow: 13px 15px 0px #802d10;
  border-radius: 43px;
  display:flex;
  flex-direction: column;
  padding: 4em 3em;
}

.login form button {
  background: #f6511d;
  border: 1px solid rgba(128, 45, 16, 0.27);
  box-sizing: border-box;
  box-shadow: 0px 4px 4px rgba(128, 45, 16, 0.35);
  border-radius: 0.7em;
  color: white;
  padding:0.5em 2em;
  max-width: 75%;
  margin: 0 auto;
  font-size: 1.3em;
  font-family: "Ropa Sans", sans-serif;
}
