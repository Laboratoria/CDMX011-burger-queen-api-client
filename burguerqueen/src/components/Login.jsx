import Logo from "./images/Logo.png";
import cocinero from "./images/cocinero.png";
import guardaespaldas from "./images/guardaespaldas.png";
import camarera from "./images/camarera.png";
import { Fragment } from "react";
import Button from "@mui/material/Button";
import React, { useState } from "react";
import TextField from "@mui/material/TextField";
import InputAdornment from "@mui/material/InputAdornment";

function Login() {
  const [email, setEmail] = useState("");
  const [password, setPassword] = useState("");

  const enviarDatos = (event) => {
    event.preventDefault();
    console.log(email, password);
  };
 
  return (
    <Fragment>
      <header>
        <img src={Logo} alt=""></img>
        <section>
          <h1>Inicia sesi칩n</h1>
        </section>
      </header>
      <main>
        <div className="Usuarios">
          <img src={guardaespaldas} alt=""></img>
          <img src={camarera} alt=""></img>
          <img src={cocinero} alt=""></img>
        </div>
        <form onSubmit={enviarDatos}>
          <TextField
            fullWidth
            autoFocus
            color="secondary"
            margin="normal"
            variant="outlined"
            name="email"
            type="email"
            label="Correo electr칩nico"
            onChange={(e) => setEmail(e.target.value)}
            required
       
            InputProps={{
              endAdornment: (
                <InputAdornment >
                  <svg
                    width="25"
                    height="24"
                    viewBox="0 0 25 24"
                    fill="none"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M22.8701 6C22.8701 4.9 21.9701 4 20.8701 4H4.87012C3.77012 4 2.87012 4.9 2.87012 6V18C2.87012 19.1 3.77012 20 4.87012 20H20.8701C21.9701 20 22.8701 19.1 22.8701 18V6ZM20.8701 6L12.8701 11L4.87012 6H20.8701ZM20.8701 18H4.87012V8L12.8701 13L20.8701 8V18Z"
                      fill="#A3A3A3"
                    />
                  </svg>
                </InputAdornment>
              ),
            }}
          />
          <TextField
            fullWidth
            color="secondary"
            margin="normal"
            variant="outlined"
            name="password"
            type="password"
            label="Contrase침a"
            onChange={(e) => setPassword(e.target.value)}
            required
         
            InputProps={{
              endAdornment: (
                <InputAdornment >
                
                  <svg
                  endAdornment
                    width="25"
                    height="24"
                    viewBox="0 0 25 24"
                    fill="none"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M2.87012 17H22.8701V19H2.87012V17ZM4.02012 12.95L4.87012 11.47L5.72012 12.95L7.02012 12.2L6.17012 10.72H7.87012V9.22H6.17012L7.02012 7.75L5.72012 7L4.87012 8.47L4.02012 7L2.72012 7.75L3.57012 9.22H1.87012V10.72H3.57012L2.72012 12.2L4.02012 12.95V12.95ZM10.7201 12.2L12.0201 12.95L12.8701 11.47L13.7201 12.95L15.0201 12.2L14.1701 10.72H15.8701V9.22H14.1701L15.0201 7.75L13.7201 7L12.8701 8.47L12.0201 7L10.7201 7.75L11.5701 9.22H9.87012V10.72H11.5701L10.7201 12.2V12.2ZM23.8701 9.22H22.1701L23.0201 7.75L21.7201 7L20.8701 8.47L20.0201 7L18.7201 7.75L19.5701 9.22H17.8701V10.72H19.5701L18.7201 12.2L20.0201 12.95L20.8701 11.47L21.7201 12.95L23.0201 12.2L22.1701 10.72H23.8701V9.22Z"
                      fill="#A3A3A3"
                    />
                  </svg>
                </InputAdornment>
              ),
            }}
          />

          <Button
            fullWidth
            id="Login"
            variant="contained"
            margin="large"
            type="submit"
          >
            Iniciar Sesi칩n
          </Button>
        </form>
      </main>
    </Fragment>
  );
}

export default Login;
