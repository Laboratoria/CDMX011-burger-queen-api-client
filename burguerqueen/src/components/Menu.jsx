import React, { Fragment } from 'react'
import MenusCointeiner from './MenusCointeiner'

const Menu = () => {
    return (
        <Fragment>
            <h1>Menu</h1>
            <MenusCointeiner />
        </Fragment>
    )
}

export default Menu
