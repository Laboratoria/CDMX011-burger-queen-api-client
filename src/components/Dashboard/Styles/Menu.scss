.menu{
    width: 55vw;
    display:flex;
    flex-wrap: wrap;
    gap: 2em;
    padding: 3em;
    align-items: flex-start;
}