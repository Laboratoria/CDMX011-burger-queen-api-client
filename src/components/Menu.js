import React, { Fragment } from 'react'
// import { Producto } from './Producto';

export const Menu = (product) => {
    // let {
    //     products
    //   } = props;
    return (
        <div>
            {product&&
            <Fragment>
        <h2>{product.nameProduct}</h2>
        <strong>{product.price}</strong>
        </Fragment>
            }
        </div>
    );
}
