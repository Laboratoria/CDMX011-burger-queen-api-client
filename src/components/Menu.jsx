import React, { Fragment } from 'react'
import MenusCointeiner from './MenusCointeiner'
import { getUser } from '../lib/FirebaseAut'

const Menu = () => {
    const {email}= getUser()
    return (
        <Fragment >
            <h1 id='MenuH1'>Menu </h1>
            <h2 align='center' color='secondary'>Bienvenido {email}</h2>
            <MenusCointeiner />
        </Fragment>
    )
}

export default Menu
