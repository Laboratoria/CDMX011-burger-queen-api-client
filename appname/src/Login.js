import React, { useState } from "react";
import { useNavigate, Link } from "react-router-dom";
import { signInWithEmailAndPassword } from "firebase/auth";
//import { useAuthState } from "react-firebase-hooks/auth";
import auth from "./firebase.js"
import "./Login.css";

function Login() {
  const [email, setEmail] = useState("");
  const [password, setPassword] = useState("");
  const [error, setError] = useState("");
 // const [error] = useAuthState(getAuth);
  const history = useNavigate();

  //const auth = getAuth();

  const signIn = (email, password) => {
    signInWithEmailAndPassword(auth, email, password)
      .then(() => {
        console.log("Sí jalo")
        history('/dashboard');
      })
      .catch(() => {
        console.log("No jalo")
        setError('Contraseña y/o correo inválidos');
        console.log(error,setError);
      });
  };
  /*     useEffect(() => {
        if(loading) {

            return;
        }
        if(user) history.replace("/dashboard");
    }, [user, loading]); */
  return (
    <main className="login">
      <form className="login__container">
        <label >Email</label>
        <input
          id="email"
          type="email"
          className="login__textBox"
          value={email}
          onChange={(e) => setEmail(e.target.value)}
          placeholder="Escribe tu correo"
        />
        <label >Contraseña</label>
        <input
          id="password"
          type="password"
          className="login__textBox"
          value={password}
          onChange={(e) => setPassword(e.target.value)}
          placeholder="Escribe tu contraseña"
        />
        <Link to="/dashboard">
        <button
          className="login__btn"
          onClick={() => signIn(email, password)}
        >
          Acceder
        </button>
        </Link>
      </form>
    </main>
  );
}

export default Login;
