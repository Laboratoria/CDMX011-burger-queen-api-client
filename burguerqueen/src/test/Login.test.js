import React, { useState } from "react";
import Login from "../components/Login";
import Form from "../components/Form";
import Menu from "../components/Menu";
import { render, fireEvent, screen} from '@testing-library/react'



test('<Renders the menu page with the correct data', () => {
 

  const component = render(
    <Form  />
  )

  const email = component.container.querySelector('#email')
  fireEvent.change(email, { 
    target: { value: 'correo@ejemplo.com' } 
  })
  const password = component.container.querySelector('#password')
  fireEvent.change(password, { 
    target: { value: '123456' } 
  })
  const button = component.container.querySelector('#Login')
  fireEvent.click(button)
  expect(email.value).toBe('correo@ejemplo.com')
  expect(password.value).toBe('123456')
  expect(render(<Menu />)).toMatchSnapshot()
  expect(render(<Menu />).container.textContent).toBe("Menu");
 



})
test('<Renders the same page with the incorrect data', () => {
 

  const component = render(
    <Form />
  )

  const email = component.container.querySelector('#email')
  fireEvent.change(email, { 
    target: { value: 'correo1@ejemplo.com' } 
  })
  const password = component.container.querySelector('#password')
  fireEvent.change(password, { 
    target: { value: '1234567' } 
  })
  const button = component.container.querySelector('#Login')
  fireEvent.click(button)
  expect(email.value).toBe('correo1@ejemplo.com')
  expect(password.value).toBe('1234567')
  expect(render(<Login />)).toMatchSnapshot()


})

 test('<Icon Should show on should password', () => {
 

  const component = render(
    <Form  />
  )
const password = component.container.querySelector('#password')
  const show = component.container.querySelector('#show')

  const unshow = component.container.querySelector('#unShow')

 
  fireEvent.click(show)
  expect(password.type).toBe('password')
  fireEvent.click(unshow)
  expect(password.type).toBe('text')
  
 



}) 

