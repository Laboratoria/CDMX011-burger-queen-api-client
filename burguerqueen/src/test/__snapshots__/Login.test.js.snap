// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Renders the menu page with the correct data 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <form
        class="form"
        data-testid="form"
        id="form"
      >
        <h1>
          Inicia Sesión
        </h1>
        <div
          class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
        >
          <label
            class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorSecondary MuiFormLabel-filled Mui-focused Mui-required css-5qrpso-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink="true"
            for="email"
            id="email-label"
          >
            Correo electrónico
            <span
              aria-hidden="true"
              class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorSecondary MuiInputBase-fullWidth Mui-focused MuiInputBase-formControl MuiInputBase-adornedEnd css-1wnwmuc-MuiInputBase-root-MuiOutlinedInput-root"
          >
            <input
              aria-invalid="false"
              class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
              id="email"
              required=""
              type="email"
              value="correo@ejemplo.com"
            />
            <div
              class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-1laqsz7-MuiInputAdornment-root"
              id="show"
            >
              <img
                src="./images/emailIcon.svg"
              />
            </div>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-186xcr5"
              >
                <span>
                  Correo electrónico
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
        >
          <label
            class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorSecondary MuiFormLabel-filled Mui-required css-5qrpso-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink="true"
            for="password"
            id="password-label"
          >
            Contraseña
            <span
              aria-hidden="true"
              class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorSecondary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd css-1wnwmuc-MuiInputBase-root-MuiOutlinedInput-root"
          >
            <input
              aria-invalid="false"
              class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
              id="password"
              required=""
              type="password"
              value="123456"
            />
            <div
              class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-1laqsz7-MuiInputAdornment-root"
              id="unShow"
            >
              <img
                src="./images/iconoNotPassword.svg"
              />
            </div>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-186xcr5"
              >
                <span>
                  Contraseña
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <button
          class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButtonBase-root css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
          data-testid="Login"
          id="Login"
          margin="large"
          tabindex="0"
          type="submit"
        >
          Iniciar Sesión
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </form>
    </div>
    <div>
      <h1>
        Menu
      </h1>
    </div>
  </body>,
  "container": <div>
    <h1>
      Menu
    </h1>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`<Renders the same page with the incorrect data 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <form
        class="form"
        data-testid="form"
        id="form"
      >
        <h1>
          Inicia Sesión
        </h1>
        <div
          class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
        >
          <label
            class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorSecondary MuiFormLabel-filled Mui-required css-5qrpso-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink="true"
            for="email"
            id="email-label"
          >
            Correo electrónico
            <span
              aria-hidden="true"
              class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorSecondary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd css-1wnwmuc-MuiInputBase-root-MuiOutlinedInput-root"
          >
            <input
              aria-invalid="false"
              class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
              id="email"
              required=""
              type="email"
              value="correo1@ejemplo.com"
            />
            <div
              class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-1laqsz7-MuiInputAdornment-root"
              id="show"
            >
              <img
                src="./images/emailIcon.svg"
              />
            </div>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-186xcr5"
              >
                <span>
                  Correo electrónico
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
        >
          <label
            class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorSecondary MuiFormLabel-filled Mui-required css-5qrpso-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink="true"
            for="password"
            id="password-label"
          >
            Contraseña
            <span
              aria-hidden="true"
              class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorSecondary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd css-1wnwmuc-MuiInputBase-root-MuiOutlinedInput-root"
          >
            <input
              aria-invalid="false"
              class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
              id="password"
              required=""
              type="password"
              value="1234567"
            />
            <div
              class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-1laqsz7-MuiInputAdornment-root"
              id="unShow"
            >
              <img
                src="./images/iconoNotPassword.svg"
              />
            </div>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-186xcr5"
              >
                <span>
                  Contraseña
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <button
          class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButtonBase-root css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
          data-testid="Login"
          id="Login"
          margin="large"
          tabindex="0"
          type="submit"
        >
          Iniciar Sesión
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </form>
    </div>
    <div>
      <main>
        <div
          class="Usuarios"
        >
          <img
            alt=""
            src="./images/guardaespaldas.png"
          />
          <img
            alt=""
            src="./images/camarera.png"
          />
          <img
            alt=""
            src="./images/cocinero.png"
          />
        </div>
        <form
          class="form"
          data-testid="form"
          id="form"
        >
          <h1>
            Inicia Sesión
          </h1>
          <div
            class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorSecondary Mui-focused Mui-required css-5qrpso-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="true"
              for="email"
              id="email-label"
            >
              Correo electrónico
              <span
                aria-hidden="true"
                class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorSecondary MuiInputBase-fullWidth Mui-focused MuiInputBase-formControl MuiInputBase-adornedEnd css-1wnwmuc-MuiInputBase-root-MuiOutlinedInput-root"
            >
              <input
                aria-invalid="false"
                class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
                id="email"
                required=""
                type="email"
                value=""
              />
              <div
                class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-1laqsz7-MuiInputAdornment-root"
                id="show"
              >
                <img
                  src="./images/emailIcon.svg"
                />
              </div>
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-186xcr5"
                >
                  <span>
                    Correo electrónico
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <div
            class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorSecondary Mui-required css-1v4qvbo-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="false"
              for="password"
              id="password-label"
            >
              Contraseña
              <span
                aria-hidden="true"
                class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorSecondary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd css-1wnwmuc-MuiInputBase-root-MuiOutlinedInput-root"
            >
              <input
                aria-invalid="false"
                class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
                id="password"
                required=""
                type="password"
                value=""
              />
              <div
                class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-1laqsz7-MuiInputAdornment-root"
                id="unShow"
              >
                <img
                  src="./images/iconoNotPassword.svg"
                />
              </div>
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-dhh0px"
                >
                  <span>
                    Contraseña
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <button
            class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButtonBase-root css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
            data-testid="Login"
            id="Login"
            margin="large"
            tabindex="0"
            type="submit"
          >
            Iniciar Sesión
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </form>
      </main>
    </div>
  </body>,
  "container": <div>
    <main>
      <div
        class="Usuarios"
      >
        <img
          alt=""
          src="./images/guardaespaldas.png"
        />
        <img
          alt=""
          src="./images/camarera.png"
        />
        <img
          alt=""
          src="./images/cocinero.png"
        />
      </div>
      <form
        class="form"
        data-testid="form"
        id="form"
      >
        <h1>
          Inicia Sesión
        </h1>
        <div
          class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
        >
          <label
            class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorSecondary Mui-focused Mui-required css-5qrpso-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink="true"
            for="email"
            id="email-label"
          >
            Correo electrónico
            <span
              aria-hidden="true"
              class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorSecondary MuiInputBase-fullWidth Mui-focused MuiInputBase-formControl MuiInputBase-adornedEnd css-1wnwmuc-MuiInputBase-root-MuiOutlinedInput-root"
          >
            <input
              aria-invalid="false"
              class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
              id="email"
              required=""
              type="email"
              value=""
            />
            <div
              class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-1laqsz7-MuiInputAdornment-root"
              id="show"
            >
              <img
                src="./images/emailIcon.svg"
              />
            </div>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-186xcr5"
              >
                <span>
                  Correo electrónico
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiFormControl-root MuiFormControl-marginNormal MuiFormControl-fullWidth MuiTextField-root css-17vbkzs-MuiFormControl-root-MuiTextField-root"
        >
          <label
            class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorSecondary Mui-required css-1v4qvbo-MuiFormLabel-root-MuiInputLabel-root"
            data-shrink="false"
            for="password"
            id="password-label"
          >
            Contraseña
            <span
              aria-hidden="true"
              class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorSecondary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd css-1wnwmuc-MuiInputBase-root-MuiOutlinedInput-root"
          >
            <input
              aria-invalid="false"
              class="MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputAdornedEnd css-nxo287-MuiInputBase-input-MuiOutlinedInput-input"
              id="password"
              required=""
              type="password"
              value=""
            />
            <div
              class="MuiInputAdornment-root MuiInputAdornment-positionEnd MuiInputAdornment-outlined MuiInputAdornment-sizeMedium css-1laqsz7-MuiInputAdornment-root"
              id="unShow"
            >
              <img
                src="./images/iconoNotPassword.svg"
              />
            </div>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-dhh0px"
              >
                <span>
                  Contraseña
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <button
          class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-fullWidth MuiButtonBase-root css-1fu7jd5-MuiButtonBase-root-MuiButton-root"
          data-testid="Login"
          id="Login"
          margin="large"
          tabindex="0"
          type="submit"
        >
          Iniciar Sesión
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </form>
    </main>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
